### DISCUSSION TITLE
Parameters in save_fits function

### DISCUSSION THREAD

**Discussion by bschneider-astro** [cylammarco/ASPIRED#78] [Ideas]
Hi @cylammarco, regarding the **output** parameter of the save_fits function, it would be interesting to have the possibility to save the resampled wavelength array corresponding to flux_resampled. I assume this information is available in the header but I think it would be nice to have an HDU with the values directly. Also, in the case of a 1D spectrum it might be interesting to save the data in an HDU table rather than HDU images, with each column of the table corresponding to flux, wavelength, flux_err and sky. Benjamin

**Comment by cylammarco** [comment]
Hi @bschneider-astro, if having a `wavelength_resampled` HDU would make life easier I can certainly add that as an option in v0.5, which is something I will start working on very soon.

Regarding HDU image and HDU table. An `image` type was chosen because in that way the output 1D spectrum can be read by iraf's `splot`. This is a minimal iraf-compatibility I am providing. Do you know much about other spectral analysis toolkits? Do they usually read both HDU image and HDU table?
